src/ray/gcs/tables.h

```diff
@@ -312,12 +376,12 @@ class ClientTable : private Table<ClientID, ClientTableData> {
 
  private:
   /// Handle a client table notification.
-  void HandleNotification(AsyncGcsClient *client, const ClientID &channel_id,
-                          const ClientTableDataT &notifications);
+  void HandleNotification(AsyncGcsClient *client, const ClientTableDataT &notifications);
   /// Handle this client's successful connection to the GCS.
-  void HandleConnected(AsyncGcsClient *client, const ClientID &client_id,
-                       const ClientTableDataT &notifications);
+  void HandleConnected(AsyncGcsClient *client, const ClientTableDataT &notifications);
 
+  /// The key to append client table entries to.
+  UniqueID client_log_key_;
```

Comment:
I'm confused by this comment.  Can you say what the key is (is nil an arbitrary choice), and what values it points to?  Might be good to write something like `client_log_key -> [client0, client1, ...]`, as a made-up example.

Reply:
The client table is a little weird because there's actually only a single log, vs the object table where there is a log per object table key. All of the clients of the client table need to read and write the same log of client entries, so they all have to know the same key. I chose nil so that the clients always know at which key the client log is stored. I'll add some documentation for this.